### ISSUE TITLE
Cannot show password

### ISSUE THREAD

**Issue by xiaoshuw** [lastpass/lastpass-cli#703]
When running the following command

lpass show x.y.z
Error: Could not unbase64 the given bytes.

My machine is: Apple M1 Pro
macOS: 15.0.1 (24A348)

**Comment by deepaktherma** [comment]
Same issue when adding an account
`printf "URL: url\nUsername: username\nPassword: password" | lpass add --non-interactive --sync=now Shared-Accounts/User
Error: Could not unbase64 the given bytes.`

**Comment by tuxlife** [comment]
I have the same issue with version 1.5.0 and 1.6.1

**Comment by xiaoshu-wang** [comment]
well, it looks like last pass is dying. No response whatsoever for three months.

**Comment by LuL1ll0** [comment]
FWIW, I cant reproduce either problem described above.

**Comment by roooneey** [comment]
yeah I'm in the same boat. I can create 5 passwords at a time then I'll get the error. Logout and login is the only workaround I found. But seriously it's ridiculous to add those kind of workarounds. I also think they are going away, it's probably better to look for an alternative.

**Comment by TheMysteriousX** [comment]
The issue seems to occur when there's a change pending in the upload queue.

I can reliably reproduce it by:
* monitor the number of files in _.lpass/upload-queue_
* run a bunch of searches (100+) to fill up the upload queue
* create/edit one note using the cli before the queue reaches zero

The error will be returned until the upload queue directory is empty. Creating several notes in short succession will fill the upload-queue too.

Unfortunately nothing is logged even with the debug value set to 8, so no clue where it is failing.

I'd suggest not logging in/out to fix it - that may cause pending uploads to be lost permanently. One the queue drops to zero the cli client should start working again. It may take a while because lastpass seems to heavily throttle the cli client to around something like 30 uploads/minute.
